File: users-ms/dtos/user_dto.py
from dataclasses import dataclass, field

@dataclass
class UserDTO:
    name: str = field(default=None)
    last_name: str = field(default=None)
    age: int = field(default=None)
    height: float = field(default=None)
    weight: float = field(default=None)
    arm: int = field(default=None)
    chest: int = field(default=None)
    waist: int = field(default=None)
    leg: int = field(default=None)
    withdrawal_date: str = field(default=None)
    withdrawal_reason: str = field(default=None)

File: web/src/app/dtos/user.dto.ts
export class UserDto {
  constructor(
    public name: string,
    public last_name: string,
    public age: number,
    public height: number,
    public weight: number,
    public arm: number,
    public chest: number,
    public waist: number,
    public leg: number,
    public created_at?: string,
    public updated_at?: string,
    public withdrawal_date?: string,
    public withdrawal_reason?: string,
    public id?: number,
  ) { }
}

export interface UserPageDto {
  users: UserDto[];
  page: number;
  per_page: number;
  total: number;
  total_pages: number;
}